addon/src/modifiers/sortable-item.js

```diff
@@ -366,6 +406,9 @@ export default class SortableItemModifier extends Modifier {
    * @private
    */
   _prepareDrag(startEvent, event) {
+    if (this.sortableGroup.sortedItems.some((x) => x.isBusy)) {
+      return;
+    }
```

Comment:
always check if any item is busy, otherwise lock dragging. There is a isBusy check in an other place, but it checks always one the current item which you want drag.
Without this check can already drag items, without finishing the transition from old element. Simple repro is, set transition time to 1 sec

Reply:
this changes were moved to PR #562